src/autoscaler/eventgenerator/generator/evaluator_test.go

```diff
@@ -613,6 +607,7 @@ var _ = Describe("Evaluator", func() {
 							errorStr = errorStr + tmp
 						}
 						scalingEngine.RouteToHandler("POST", urlPath, ghttp.RespondWithJSONEncoded(http.StatusBadRequest, errorStr))
+						Expect(triggerChan).To(BeSent(triggerArrayGT))
```

Comment:
I see.. yeah, seems odd to have a test that never runs, because (a) it's not actually going to fail so it doesn't stop stuff being broken (b) it will probably end up accidentally becoming out of date, like it seems to have here because at least for me it consistently fails right now. Maybe we should either find a way to make it faster so we can un-pend it (e.g. I think it should be possible to create the large string in a more efficient way, for example by pre-computing the string as a constant) or remove it, although I can create a separate bug/chore to do that, it doesn't need to be part of this PR, I was just wondering about it